.github/pull_request_template.md

```diff
@@ -3,11 +3,11 @@ BEFORE YOU CREATE A PULL REQUEST:
 
 Ensure you have read over
 
-CONTRIBUTING - https://github.com/lingua-pupuli/puppet-editor-services/blob/master/CONTRIBUTING.md
+CONTRIBUTING - https://github.com/puppetlabs/puppet-editor-services/blob/master/CONTRIBUTING.md
 
 and
 
-CODE_OF_CONDUCT - https://github.com/lingua-pupuli/puppet-editor-services/blob/master/CODE_OF_CONDUCT.md
+CODE_OF_CONDUCT - https://github.com/puppetlabs/puppet-editor-services/blob/master/CODE_OF_CONDUCT.md
```

Comment:
should this be replaced by https://github.com/puppetlabs/.github/blob/master/CODE_OF_CONDUCT.md ?

(there's also a CONTRIBUTING.md, but I'm not so sure about that one)

Reply:
Maybe, but should be part of another PR. Changing code of conduct and contributing is separate idea than changing links to the same content

Reply:
yeah.

Reply:
Created an issue to track the code of conduct and contributing updates. Resolving this